Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV["TERM"] = "xterm-256color"
+ENV["GOPATH"] = "/go"
+ENV["LC_ALL"] = "en_US.UTF-8"
+
+Vagrant.configure("2") do |config|
+  # config.vm.box = "fedora/25-cloud-base"
+  # config.vm.provision "shell", inline: "dnf install -y go git docker"
+  config.vm.box = "ubuntu/zesty64"
+  config.vm.provision "shell", inline: "DEBIAN_FRONTEND=noninteractive apt-get install -y golang git docker.io systemd-container tmux"
+
+  # config.vbguest.auto_update = true
+
+  config.vm.synced_folder ".", "/vagrant", disabled: true
+  config.vm.synced_folder ".", "/go/src/github.com/kinvolk/kubeadm-nspawn", create: true
+  # config.vm.provider :virtualbox do |vbox|
+  #     vbox.check_guest_additions = false
+  #     vbox.functional_vboxsf = false
+  # end
+
+  config.vm.provider :virtualbox do |vb|
+      vb.customize ["modifyvm", :id, "--memory", "1024"]
```

Comment:
I don't know either... I would use 4096 and investigate if the error ever happens again.